### ISSUE TITLE
Clicking on link in note should open Map View

### ISSUE THREAD

**Issue by dummifiedme** [esm7/obsidian-map-view#36]
Currently, on mobile, clicking the link opens up google maps(Earth if Earth is installed, maybe a problem with defaults), but I think if there is an option to open the link into Map View inside the map view itself it would be great.

Problem is real in Linux. It throws an error as I dont exactly have a "Map" app. I can open googlemaps on browser but only after I go to the marker on the map.

So, it would really be a great help if somehow the links in note open in the Map View!! — Just like notes, images or excalidraw or pdfs open in new pane when we click the links, :)

Let me know if I am not clear.

**Comment by Darkbluetest** [comment]
This is a must. Why having a map just in Obsidian, we should go to google or other? I mean, is useful, but the default option should be, imo, to show/center map in for the location clicked.

**Comment by esm7** [comment]
Until I do that (which I agree that makes a lot of sense), you can right-click the line with the link and choose to open the map view.
(Not as sleek and an extra step compared to clicking the link, I agree...)

**Comment by TheAnachronism** [comment]
Currently facing the same problem on Apple...
Just clicking on the inline geo-link opens the internal map view (cartodb). But when hovering over it the icon appears to open the link and clicking on that, opens Google Earth instead of the internal map.

**Comment by esm7** [comment]
I didn't notice this is a dup of https://github.com/esm7/obsidian-map-view/issues/144 (actually the other way around but NM).
To make sure I understand, do you mean that the link itself works correctly, but the icon next to it (the little box with the arrow going out, in the default theme) opens an external app?

**Comment by esm7** [comment]
Ok, I am able to recreate it, and I'm hopeful that the overly complicated method needed to hijack the Obsidian editor links is expandable to the icon too.
However, would you mind to open a new issue for that? I'm hoping to check it out soon, but in case I eventually won't be able to, I rather not get confused by this older issue that is actually resolved.
